pkg/client/kubernetes.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	k8yaml "sigs.k8s.io/yaml"
+)
+
+type Client struct {
+	client *kubernetes.Clientset
+}
+
+func NewClient() (*Client, error) {
+	config, err := rest.InClusterConfig()
```

Comment:
Can we have a nice error in the situation where someone tries to do this while not in kubernetes

Reply:
Added